frontend/src/app/features/auth/sign-up/sign-up.component.ts

```diff
@@ -0,0 +1,103 @@
+import { AbstractControl, FormControl, FormGroup, ValidatorFn, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { catchError, of } from 'rxjs';
+
+import { AuthService } from 'src/app/api/auth.service';
+import { Component } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { register } from 'src/app/store/actions/login-page.actions';
```

Comment:
To get rid of such imports, you may add alias in package.json to specific folders

Reply:
Fixed by adding aliases in tsconfig.json